test/chemistry/#test_molecule.py#

```diff
@@ -0,0 +1,84 @@
+import unittest
```

Comment:
This looks to me like a temporary file created by some editor. It should be removed.